bundles/config/org.eclipse.smarthome.config.xml.test/src/test/resources/test-bundle-pool/ConfigDescriptionsTest.bundle/ESH-INF/config/config.xml

```diff
@@ -8,13 +8,16 @@
 	<config-description uri="config:dummyConfig">
 
 		<parameter name="ip" type="text" pattern="[0-9]{3}.[0-9]{3}.[0-9]{3}.[0-9]{3}" required="true" readOnly="true">
+	        <multipleLimit>4</multipleLimit>
 			<context>network-address</context>
 			<label>Network Address</label>
 			<description>Network address of the hue bridge.</description>
 			<required>true</required><!-- deprecated -->
+			<limitToOptions>true</limitToOptions>
 		</parameter>
 
 		<parameter name="username" type="text">
+			<groupName>user</groupName>
```

Comment:
I would expect groupName to be an attribute and node a node. But lets discuss it.